include/solvers/vof_scratch_data.h

```diff
@@ -278,9 +278,9 @@ class VOFScratchData
         std::vector<double> time_vector =
           this->simulation_control->get_simulation_times();
         bdf_extrapolate(time_vector,
-                        previous_velocity_values,
+                        this->previous_velocity_values,
```

Comment:
Out of curiosity, is it absolutely necessary here or just a matter of style?

Reply:
Matter of style here :). It's often good to be explicit though about what is a class member and what is not :)

Reply:
Thanks !